website/docs/guides/fusion-package-compat.md

```diff
@@ -0,0 +1,169 @@
+
+A users stores their package in a `packages.yml` or `dependencies.yml` file. If a package excludes `2.0.0`, <Constant name="fusion"/> will warn today and error in a future release, matching <Constant name="core"/> behavior. 
+
+This guide assumes you're using the command line and Git to make changes in your package repository. If you're interested in creating a new package from scratch, we recommend using the [dbt package guide](/guides/building-packages) to get started.
+
+## Prerequisites
+
+Before you begin, make sure you meet the following:
+
+- dbt package maintainer &mdash; You're a dbt package maintainer with a package on [dbt package hub](https://hub.getdbt.com/) or interested in creating a new package.
+- `dbt-autofix` installed &mdash; [Install `dbt-autofix`](https://github.com/dbt-labs/dbt-autofix?tab=readme-ov-file#installation) to automatically update the package's YAML files to align with the latest dbt updates and best practices. We recommend [using/installing uv/uvx](https://docs.astral.sh/uv/getting-started/installation/) to run the tool.
+  - Run the command `uvx dbt-autofix` for the latest version of the tool. For more installation options, see the [official `dbt-autofix` doc](https://github.com/dbt-labs/dbt-autofix?tab=readme-ov-file#installation).
+- Repository access &mdash; You’ll need permission to create a branch and release updates/a new version of your package. You’ll need to tag a new version of your package once it’s <Constant name="fusion"/>-compatible.
+- A <Constant name="fusion"/> installation or test environment &mdash; You can use <Constant name="fusion"/> locally (using the `dbtf` binary) or in your CI pipeline to validate compatibility.
+- You use the command line and Git to make changes in your package repository.
+
+## Upgrade the package
+This section of the guide will go over how to upgrade your package to be compatible with <Constant name="fusion"/> by:
+- Using `dbt-autofix` to automatically update your YAML files
+- Testing your package with <Constant name="fusion"/>
+- Updating your `require-dbt-version` config
+- Publishing a new release of your package
+
+### Run dbt-autofix
+
+1. In your dbt package repository, create a branch to work in. For example:
+    ```bash
+    git checkout -b fusion-compat
+    ```
+
+2. Ensure you have `dbt-autofix` installed. If you don't have it installed, run the command `uvx dbt-autofix`. For more installation options, see the [official `dbt-autofix` doc](https://github.com/dbt-labs/dbt-autofix?tab=readme-ov-file#installation).
+
+3. Run `dbt-autofix` in your package directory so it automatically updates your package code and rewrites YAML to conform to the latest JSON schema:
+    ```bash
+    dbt-autofix deprecations
+    ```
+
+4. (Optional) You can also run `dbt-autofix` as a temporary continuous integration (CI) check until the package is <Constant name="fusion"/>-compatible:
```

Comment:
I don't understand what this means. If I ran autofix in step 3 then why would running it in CI pick anything else up? Is this intended to prevent regressions? If so then why is it only a temporary check?

Reply:
i took this from @dataders 's amazing write up [here](https://github.com/dbt-labs/dbt-fusion/discussions/401) -- Anders did i misinterpret what you wrote? should i just remove this ci check step?